pkg/runtest/run.go

```diff
@@ -204,7 +210,15 @@ func progFileList(dir, filter string) ([]string, error) {
 func (ctx *Context) generateFile(sandboxes []string, cover []bool, filename string) error {
 	p, requires, results, err := parseProg(ctx.Target, ctx.Dir, filename, nil)
 	if err != nil {
-		return err
+		// Treat non-parseable programs as failing.
+		for _, sandbox := range sandboxes {
```

Comment:
We can create just 1 failing test, we don't need to loop over sandboxes, we did not use sandbox to parse the program.

Reply:
Fair enough.